### ISSUE TITLE
[4.0] [Workflow] New Workflow breaks articles when multilingual and associations ON

### ISSUE THREAD

**Issue by infograf768** [joomla/joomla-cms#21475]
### Steps to reproduce the issue
Install 4.0-dev. Install some languages. Install the multilingual sample data.

### Bug

`500 'v40_multi.wa.state_id' isn't in GROUP BY`
articles Manager
`0 Argument 1 passed to Joomla\Utilities\ArrayHelpergetColumn() must be of the type array, boolean given, called in /administrator/components/com_content/Model/ArticlesModel.php on line 483`

### basic solution proposed by @rdeutz 
in ArticlesModel line 286ff (I love the $bingo variable 👍 )
```
		// Join over the associations.
		if (Associations::isEnabled())
		{
			$bingo = $associationsGroupBy;
   			$bingo[] = 'wa.state_id';
   			$bingo[] = 'ws.title';
   			$bingo[] = 'ws.condition';
   			$bingo[] = 'ws.workflow_id';
			
			$query->select('COUNT(asso2.id)>1 as association')
				->join('LEFT', '#__associations AS asso ON asso.id = a.id AND asso.context=' . $db->quote('com_content.item'))
				->join('LEFT', '#__associations AS asso2 ON asso2.key = asso.key')
				->group($db->quoteName($bingo));
		}
```
It works here.

### Additional comments
I do not understand the new Table Options: Select State and Select Condition.

@bembelimen

**Comment by brianteeman** [comment]
Workflow is simply not ready - see #21476

**Comment by ghost** [comment]
After clean Install and Multilanguage Sample Data get: ` Expression #13 of SELECT list is not in GROUP BY clause and contains nonaggregated column '.wa.state_id' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by `

**Comment by rdeutz** [comment]
@franz-wohlkoenig its the same problem

**Comment by ghost** [comment]
@rdeutz wrote this because of [Comment](https://github.com/joomla/joomla-cms/issues/21474#issuecomment-411701344) and haven't linked back but will do.

**Comment by ghost** [comment]
closed as having Pull Request #21512
